handout/redux/configuring_your_application_to_use_redux.md

```diff
@@ -9,19 +9,30 @@ Angular 2 application to make use of Ng2-Redux. For this, we will need to:
 
 ## Registering Ng2-Redux with Angular 2
 
-_app/boot.ts_
+_app/index.ts_
 ```typescript
-import { bootstrap } from '@angular/platform-browser-dynmic'
-import { SimpleRedux } from './containers/app-container'
-import { NgRedux } from 'ng2-redux'
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { NgReduxModule, NgRedux } from 'ng2-redux';
+import { SimpleRedux } from './containers/app-container';
 
-bootstrap(
-  SimpleRedux,
-  [ NgRedux, /* ... */ ]);
+@NgModule({
```

Comment:
Follow the style in the [ng2-redux readme](https://github.com/angular-redux/ng2-redux/blob/master/README.md)